src/app/myworkflows/sidebar-accordion/sidebar-accordion.component.html

```diff
@@ -42,56 +42,40 @@
-              </p>
-            </mat-list-item>
-          </mat-list>
+          <ng-template *ngTemplateOutlet="items; context: { items: orgObj.published, itemState: 'published' }"></ng-template>
         </mat-tab>
         <mat-tab label="Unpublished">
-          <mat-list class="org-accordion-list">
-            <mat-list-item
-              *ngFor="let workflowObj of orgObj?.unpublished; trackBy: trackByWorkflowId"
-              [ngClass]="{
-                selectedWorkflow: (isTool$ | async) === false && (workflowId$ | async) === workflowObj?.id,
-                selectedTool: (isTool$ | async) && (workflowId$ | async) === workflowObj?.id
-              }"
-            >
-              <mat-icon class="arrow mat-icon-sm">arrow_forward</mat-icon>
-              <a
-                class="no-wrap"
-                title="{{ workflowObj?.repository + (workflowObj?.workflowName ? '/' + workflowObj?.workflowName : '') }}"
-                [routerLink]="'/my-' + (entryType$ | async) + 's/' + workflowObj.full_workflow_path"
-              >
-                {{ workflowObj?.repository + (workflowObj?.workflowName ? '/' + workflowObj?.workflowName : '') }}
-              </a>
-            </mat-list-item>
-            <mat-list-item *ngIf="!orgObj || orgObj?.unpublished.length === 0">
-              <p [attr.data-cy]="'no-unpublished-' + pageEntryType + '-message'" class="text-center size-small">
-                No unpublished {{ pageEntryType === EntryType.AppTool ? 'tool' : pageEntryType }}s.
-              </p>
-            </mat-list-item>
-          </mat-list>
+          <ng-template *ngTemplateOutlet="items; context: { items: orgObj.unpublished, itemState: 'unpublished' }"></ng-template>
+        </mat-tab>
+        <mat-tab *ngIf="orgObj?.archived.length" label="Archived">
+          <ng-template *ngTemplateOutlet="items; context: { items: orgObj.archived, itemState: 'archived' }"></ng-template>
         </mat-tab>
       </mat-tab-group>
+      <ng-template #items let-items="items" let-itemState="itemState">
+        <mat-list class="org-accordion-list">
+          <mat-list-item
+            *ngFor="let workflowObj of items; trackBy: trackByWorkflowId"
+            [ngClass]="{
+              selectedWorkflow: (isTool$ | async) === false && (workflowId$ | async) === workflowObj?.id,
+              selectedTool: (isTool$ | async) && (workflowId$ | async) === workflowObj?.id
+            }"
+          >
+            <mat-icon class="arrow mat-icon-sm">arrow_forward</mat-icon>
+            <a
+              class="no-wrap"
+              title="{{ workflowObj?.repository + (workflowObj?.workflowName ? '/' + workflowObj?.workflowName : '') }}"
+              [routerLink]="'/my-' + (entryType$ | async) + 's/' + workflowObj.full_workflow_path"
+            >
+              {{ workflowObj?.repository + (workflowObj?.workflowName ? '/' + workflowObj?.workflowName : '') }}
```

Comment:
Ditto about the pipe https://github.com/dockstore/dockstore-ui2/blob/365875d0ddd27c04f988185e84fe9e284ed30c3f/src/app/shared/entry-to-display-name.pipe.ts#L11